images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
I know, in principle I'm wondering why we need a special ruby for MIQ. There are plenty of ruby containers already, need to check SCL offerings.